### PR TITLE
[delegation] address mainnet testing feedback

### PR DESCRIPTION
**Pull request by jillxuu**
1. make staking and unstaking warning message more accurate.
    - if active pool has less than 10 apt after txn, unlock all
    - if pending_inactive pool has less than 10 apt after txn
    - if active pool has enough stake, unlock 10 apt to meet minimum requirement, else unlock all
2. update tooltip wording on apy, and staking status
3. update validator state to reflect all 4 states
4. fix the txn succeeded dialog amount mismatch issue: https://aptos-org.slack.com/archives/C04EXV6UTR7/p1681942876524159
5. fix withdraw action when validator's inactive to check for both withdraw and pending_inactive fund 

Screenshot 2023-04-19 at 5 00 13 PM
Screenshot 2023-04-19 at 4 59 56 PM
Screenshot 2023-04-19 at 5 00 17 PM

| before | after |
| -- | -- |
| Screenshot 2023-04-19 at 5 01 31 PM | Screenshot 2023-04-19 at 5 00 46 PM |

### CODE REVIEW

**Comment by geekflyer** [comment]
the Deposit button is broken in testnet. Getting this error:
image

**Review by 0xZihan — APPROVED** [review]

**Comment by 0xZihan** [comment]
Accept to unblock. Please take care of the error before landing.

**Review by movekevin — APPROVED** [review]
Can we add a tooltip to explain why when you withdraw < 10 (and the pending inactive pool has < 10), 10 APT will be withdrawn instead?

**Pull request merged**
